masschallenge/django-accelerator
# [AC-6494] Build the new Side Navigation in a Django Template

### Changes introduced in [AC-6494](https://masschallenge.atlassian.net/browse/AC-6494):
- Add data migration to create side nav tree
- Add data migration to add items to the side nav tree

### DISCLAIMER!
- This has an accompanying PR in [accelerate](https://github.com/masschallenge/accelerate/pull/2095)
- This should be merged before the Accelerate PR

### How to test
- While checked out to this branch, run `make migrate` in impact
- Once the migration is done visit [NavTree in admin](http://localhost:8181/admin/mc/navtree/)
- You should see a `mc_side_nav_tree` with the required items as per the ticket
